### PR TITLE
Update moa quickstart quota, logs, and moactl login

### PR DESCRIPTION
**Pull request by lamek**
Updates to the moactl quicktart:

- Update moactl login to the default command (output provides more context)
- Update the logs command to use the correct syntax
- Update the quota check to explicitly specify a region.

### CODE REVIEW

**Comment by lamek** [comment]
Manual preview - http://file.bos.redhat.com/klamenzo/moa/openshift-moa/moa/getting_started_moa/creating-first-moa-cluster.html

File: getting_started_moa/creating-first-moa-cluster.adoc [hunk 1]
```diff
@@ -18,7 +18,7 @@ $ moactl init
 $ moactl create cluster --name <my-cluster-name>
 
 ## Watch your logs as your cluster creates
-$ moactl logs cluster <my-cluster-name> --watch
+$ moactl logs install <my-cluster-name> --watch
```
Comment (adellape) [inline getting_started_moa/creating-first-moa-cluster.adoc:21]: Replaceables should [use underscores](https://github.com/openshift/openshift-docs/blob/master/contributing_to_docs/doc_guidelines.adoc#code-blocks-command-syntax-and-example-output).

**Review by adellape — APPROVED** [review]
LGTM! Just a minor nit.

File: getting_started_moa/creating-first-moa-cluster.adoc (hunk 1 above)
Comment (lamek) [reply getting_started_moa/creating-first-moa-cluster.adoc:21]: I'm gonna do a "fast-follow" PR and replace them all at once. I used "-" throughout.

**Pull request merged**